### DISCUSSION TITLE
Multiple viewports

### DISCUSSION THREAD

**Discussion by mohandarsi** [raysan5/raylib#2345] [Q&A] [answered]
Is it possible to create an application that can show multiple viewports with raylib??
I could not find any example related to this in GitHub of raylib. Appreciate it if anyone provides some hints on this?? 
Here is an simple example.

![Multple views ](https://koirenderer.files.wordpress.com/2015/12/viewport.png)

**Comment by raysan5** [comment, marked as answer]
@mohandarsi You can accomplish that using 4 RenderTextures, render each 3d view in a separate RenderTexture and then draw all them to the main framebuffer.

**Reply by RonenNess** [reply]
I stumbled upon this when searching for an answer myself and I see you recommend render targets, but I also see there's this method rlViewport() that works pretty well and does exactly what @mohandarsi need.

Is there a reason not to use rlViewport() for this case?

**Reply by raysan5** [reply]
`rlViewport()` is part of `rlgl` and it's not exposed by `raylib` by default, neither from bindings to other languages.
